migrate/v0_26/migrate.md

```diff
@@ -0,0 +1,77 @@
+# Kava 16 Upgrade Instructions
+
+## Software Version and Key Dates
+
+- The version of `kava` for Kava 16 is `v0.26.0`
+- The Kava 16 chain will be shutdown with a `SoftwareUpgradeProposal` that
+  activates at approximately 15:00 UTC on April 27th, 2024.
```

Comment:
Is this a typo? Do we mean April 24th?  Or is this a placeholder?

Reply:
Yeah placeholder

Reply:
will change to TBD